src/dlr_treelite.cc

```diff
@@ -208,6 +208,11 @@ void TreeliteModel::GetOutput(int index, void* out) {
       sizeof(float) * (treelite_input_->num_row) * treelite_output_size_);
 }
 
+const void* TreeliteModel::GetOutputPtr(int index) const {
+  CHECK(treelite_input_);
```

Comment:
I copied it from `GetOutput` method.

`dlr_treelite.h` defines the following
```
  std::unique_ptr<TreeliteInput> treelite_input_;
  std::vector<float> treelite_output_;
```
I guess `CHECK` just validates that `treelite_input_` is not null because `GetOutput` needs `treelite_input_->num_row`.

`treelite_output_` is not a pointer and can not be null.
BTW, we can check if `treelite_output_.size() > 0` and `CHECK(treelite_output_.data())`. What you think?

Reply:
I see, the current way is fine then